### PR TITLE
TP2000-614 add version control tab to the view commodities details page

### PR DESCRIPTION
**Pull request by dalecannon**
# TP2000-614 add version control tab to the view commodities details page

## Why
The version control tab is a useful way to display the version history of an object in TAP.  This is currently missing on Commodities detail pages and Quotas detail pages (TP2000-649). 

## What
Creates and uses a common version control template where possible
Adds version control tab to Commodities details page
Adds version control tab to Quotas details page
Fixes 'End Date' on version control tab for Measures details pages 

## Checklist
- Requires migrations? No
- Requires dependency updates? No

Screenshot 2022-12-08 at 10 15 52
Screenshot 2022-12-08 at 16 55 05
Screenshot 2022-12-08 at 10 17 36

### CODE REVIEW

**Comment by gabelton** [comment]
Did you commit the common version_control template as part of this PR?

File: quotas/jinja2/includes/quotas/tabs/version_control.jinja [hunk 1]
```diff
@@ -9,7 +9,6 @@
     {"text": object.transaction.workbasket.get_status_display()},
     {"text": "{:%d %b %Y}".format(object.valid_between.lower)},
     {"text": "{:%d %b %Y}".format(object.valid_between.upper) if object.valid_between.upper else "-"},
-    {"text": object.get_description().description},
```
Comment (gabelton) [inline quotas/jinja2/includes/quotas/tabs/version_control.jinja:12]: Do we not want this anymore?

Comment (dalecannon) [reply quotas/jinja2/includes/quotas/tabs/version_control.jinja:12]: Not for Quotas and Measures (which still has its own template file) detail pages as they don't have a description to display, unlike the other details pages which do and will use the common version_control template.

**Comment by dalecannon** [comment]
> Did you commit the common version_control template as part of this PR?

It's there, but for some reason it's showing as a renamed and relocated additional codes version_control template.
(https://github.com/uktrade/tamato/pull/744/files#diff-4b9f1f04f1ad33f32670ec1413e83bb1309c7e7bba75d9053aa03753c438d665)

**Review by gabelton — APPROVED** [review]

**Pull request merged**